### PR TITLE
extended spec

### PR DESCRIPTION
**Pull request by dosumis**
Needed for loading neuron : neuropil connectivity with counts for Hemibrain 1.1

### CODE REVIEW

**Comment by dosumis** [comment]
@admclachlan  - Please review description text

This needs to be merged into curation branches for loading neuron : neuropil connectivity with counts for Hemibrain 1.1

File: records/new_metadata/newmeta_spec.yaml [hunk 1]
```diff
@@ -57,6 +57,23 @@ pub:
   type: pub
   edge_annotation: true
 
+upstream:
+  description: Number of postsynaptic densities on a neuron in a specific region. Use this to annotate has_presynaptic_terminal_in relationships.
```
Comment (admclachlan) [inline records/new_metadata/newmeta_spec.yaml]: 'Number of postsynaptic densities **(PSDs)** **annotated** on a neuron in a specific region.'?

It's not strictly true that the number reported is the same as the number actually there, but maybe that's not helpful to point out?

'Use this to annotate has_**post**synaptic_terminal_in relationships.'

Unless I'm misunderstanding what the relationship is, should be postsynaptic no?

File: records/new_metadata/newmeta_spec.yaml [hunk 2]
```diff
@@ -57,6 +57,23 @@ pub:
...
+  type: int
+  edge_annotation: true
+
+downstream:
+  description: Number of downstream synaptic connections (PSDs postsynaptic to the neuron T-bars) on a neuron in some specified region.  Use this to annotate 'has_postynaptic_terminal_in' relationships.
```
Comment (admclachlan) [inline records/new_metadata/newmeta_spec.yaml]: 'Number of **annotated** downstream synaptic connections (number of PSDs postsynaptic to the neuron T-bars) on a neuron in **a specific** region.'?

Make annotated status explicit and change to 'in a specific' to match the description for 'upstream' above?

'Use this to annotate 'has_**pre**ynaptic_terminal_in' relationships.'

I think I'm misunderstanding these but should it not be pre here?

File: records/new_metadata/newmeta_spec.yaml [hunk 3]
```diff
@@ -57,6 +57,23 @@ pub:
...
+  type: int
+  edge_annotation: true
+
+Tbars:
+  description: Number of Tbars that a neuron has in some specified region. Use this to annotate 'has_postynaptic_terminal_in' relationships.
```
Comment (admclachlan) [inline records/new_metadata/newmeta_spec.yaml]: 'Number of **T-bars** **annotated** on a neuron in **a specific** region. Use this to annotate 'has_**pre**ynaptic_terminal_in' relationships.'

Isn't 'T-bar' most commonly used in the literature? FBbt and GO have 't-bar' and 'T-bar' respectively. Also pre/post again, T-bars mark presynaptic sites no?

File: records/new_metadata/newmeta_spec.yaml (hunk 1 above)
Comment (dosumis) [reply records/new_metadata/newmeta_spec.yaml]: Fixed. See latest commit.

**Pull request merged**